yt_dlp/extractor/twitter.py

```diff
@@ -17,6 +19,7 @@
     make_archive_id,
     str_or_none,
     strip_or_none,
+    join_nonempty,
```

Comment:
This import is not sorted?

Reply:
Sry, my mistake while solving the conflict with my PR